.env.integration

```diff
@@ -0,0 +1,36 @@
+BUGZILLA_API_KEY=
```

Comment:
Do we need another whole file? What was wrong with the ```cp``` command?

Reply:
Yeah so the specific problem this is trying to solve is that in the tests, one of the fields that gets filled is the data science bugzilla url, and because we strongly validate that the URL provided matches the host name in the settings, in the test it's written as `http://bugzilla.com/whatever` so we need `http://bugzilla.com/` to appear as the BUGZILLA_HOST in the settings, so this way we can maintain a separate environment file in case we end up having to hardcode any other specific settings for it (like email domains or something if we ever start to include those in the tests).